scripts/setup_linux.sh

```diff
@@ -102,6 +102,11 @@ print_warning() {
   printf "\e[0;33m%s\e[0m" "${1}"
 }
 
+# Verify this script is running from the correct folder (root directory)
+([ -e setup.py ] && (more setup.py | grep -q "name='rlgarage'")) \
```

Comment:
* Please follow the [Google Shell Style Guide](https://google.github.io/styleguide/shell.xml), as mentioned in CONTRIBUTING.md.
* There's no need to open `setup.py` with `more` and pipe to `grep`. `grep` can access the file directly (see `man grep`)